docs/04-tutorial.md

```diff
@@ -127,6 +127,33 @@ rollup --config rollup.config.dev.js
 rollup --config rollup.config.prod.js
 ```
 
+### Using Rollup locally
+
+When working with multiple environments or in a team, it's often smart to add Rollup as a _local_ dependency. This way you don't have to remember (or remind team members) to install Rollup globally and you can make sure everyone is using the same version.
+
+To install Rollup locally you can simply do:
+
+```console
+npm i -D rollup
+// OR yarn -D add rollup
+```
+
+Now you can run it inside the root folder of your project like so:
+
+```console
+./node_modules/.bin/rollup --config
+```
+
+In this case it's best to just add this command as a script to your `package.json`:
+
+```json
+{
+  "scripts": {
+    "build": "./node_modules/.bin/rollup --config"
```

Comment:
This is not right. npm and yarn will resolve bin for you. Direct path will stop work with yarn v2.

Reply:
@TrySound Can you please update the PR with the correct information? I really want to know how to use Rollup locally both with NPM and Yarn.

Currently what I wrote I got from [this post](https://github.com/rollup/rollup/issues/2720#issuecomment-467357816) and they asked me to make the PR on the docs.

Currently there's no mention in the docs anywhere how to use a local rollup installation.

Reply:
The way is identical for any package with cli
```
yarn add package
yarn package
npm install package
npm run package
```

https://yarnpkg.com/en/docs/cli/#locally-installed-clis